vumi/transports/smpp/processors/default.py

```diff
@@ -470,6 +470,10 @@ class SubmitShortMessageProcessorConfig(Config):
         "If `True`, messages longer than 140 bytes will be sent as a series "
         "of smaller messages with the user data headers. Default is `False`.",
         default=False, static=True)
+    multipart_sar_reference_rollover = ConfigInt(
+        "The maximum value to set for the reference number of a multi part "
+        "SMS. When this value is reached, the number will rollover to 0.",
+        default=0xFFFF, static=True)
```

Comment:
Is a bitmask and a maximum value not the same? If I set a maximum of `255`, I would expect it to go  0 -> 255, and then rollover. I wouldn't expect it to only go to 254.